src/main/java/seedu/address/logic/parser/DeleteClientCommandParser.java

```diff
@@ -29,8 +29,7 @@ public DeleteClientCommand parse(String args) throws ParseException {
         try {
             index = ParserUtil.parseIndex(argumentMultimap.getValue(PREFIX_INDEX).get());
         } catch (ParseException pe) {
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteClientCommand.MESSAGE_USAGE), pe);
+            throw pe;
```

Comment:
there is no need to catch and throw the same error actually, should work the same without try caatch since the error should be thrown to user